dbt/loader.py

```diff
@@ -23,7 +24,17 @@ def load_all(cls, root_project, all_projects):
                     loader.load_all(root_project, all_projects, macros))
 
             to_return[subgraph] = subgraph_nodes
-
+        
+        model_fqns = []
+        for unique_id, node in to_return['nodes'].items():
+            model_fqns.append(node['fqn'])
+        
+        project_model_config_pqns = []
+        for project_name, project in all_projects.items():
+            project_model_config_pqns.extend(dbt.config.get_project_model_config_pqns(project))
```

Comment:
I've updated this bit of code to only read the `root_project` instead of `all_projects`. 

I think the helper function is still required because it's un-nesting the config into the list of lists.